index.html

```diff
@@ -168,31 +168,41 @@ <h1 class="main-text">About Myself</h1>
         </section>
     </section>
     <section class="white-bg">
-        <section class="tools">
-            <div class="languages">
-                <ul class="coder">
-                    <li><a href="#"><img src="images/languages.png" alt="languages"></a></li>
-                    <li>
-                        <ul class="lang-container">
-                            <li class="lang">
-                                <a href="#"><img class="lang-logo" src="images/javasript-logo.png" alt="javascript"></a>
-                                <p class="upper">Javascript</p>
-                            </li>
-                            <li class="lang">
-                                <a href="#"><img class="lang-logo" src="images/html-logo.png" alt="html"></a>
-                                <p class="upper">Html</p>
-                            </li>
-                            <li class="lang">
-                                <a href="#"><img class="lang-logo" src="images/css-logo.png" alt="css"></a>
-                                <p class="upper">css</p>
-                            </li>
-                        </ul>
-                    </li>
-                    <li><a href="#"><img src="images/frameworks.png" alt="frameworks"></a></li>
-                    <li><a href="#"><img src="images/skills.png" alt="skills"></a></li>
-                </ul>
-            </div>
-        </section>
-    </section> 
-</body>
+        <div class="languages">
+            <ul class="coder">
+                <li><a href="#"><img src="images/languages.png" alt="languages"></a></li>
+                <li>
+                    <ul class="lang-container">
+                        <li class="lang">
+                            <a href="#"><img class="lang-logo" src="images/javasript-logo.png" alt="javascript"></a>
+                            <p class="upper">Javascript</p>
+                        </li>
+                        <li class="lang">
+                            <a href="#"><img class="lang-logo" src="images/html-logo.png" alt="html"></a>
+                            <p class="upper">Html</p>
+                        </li>
+                        <li class="lang">
+                            <a href="#"><img class="lang-logo" src="images/css-logo.png" alt="css"></a>
+                            <p class="upper">css</p>
+                        </li>
+                    </ul>
+                </li>
+                <li><a href="#"><img src="images/frameworks.png" alt="frameworks"></a></li>
+                <li><a href="#"><img src="images/skills.png" alt="skills"></a></li>
+            </ul>
+        </div>
+    </section>
+    <section class="my-form">
```

Comment:
- [ ] You have done a great job on your form so far. however, it could be nicer to fix the issue on the top-left `border-radius`, because for now, it does not look like the one in the Figma template, see the screenshots below

- Your contact form
![image](https://user-images.githubusercontent.com/55465133/129773643-75b39953-62a9-4f30-964f-deec8f5e1db0.png)

- The Figma design contact form
![image](https://user-images.githubusercontent.com/55465133/129773768-4f948c28-44e0-44de-827b-b6ac04e4731e.png)

can you see the difference? please let us fix it.